doc/requirements.txt

```diff
@@ -29,5 +29,5 @@ sphinxcontrib-htmlhelp==2.0.1
 sphinxcontrib-jsmath==1.0.1
 sphinxcontrib-qthelp==1.0.3
 sphinxcontrib-serializinghtml==1.1.5
-urllib3>=2.2.2
-zipp==3.19.1
```

Comment:
do i understand it right, that we downgrade the version of components used?

Reply:
@emmwalsh @Maria1Petrova can you please check?

Reply:
shouldn't be the case, perhaps opensource has already newer versions, can we keep new ones?